src/components/views/settings/DevicesPanel.tsx

```diff
@@ -52,13 +53,22 @@ export default class DevicesPanel extends React.Component<IProps, IState> {
     }
 
     public componentDidMount(): void {
+        const cli = MatrixClientPeg.get();
```

Comment:
Can we use the MatrixClient from React Context please? MatrixClientPeg isn't the future

Reply:
Do you want me to refactor the rest of the class to use context instead of peg?

Reply:
That is optional, but it'd be good to not make the tech debt of using it worse

Reply:
Refactored.